.github/workflows/publish.yml

```
@@ -1,29 +1,85 @@
-name: brew pr-pull
+name: Publish Bottles
 on:
-  pull_request_target:
-    types:
-      - labeled
+  pull_request:
+    paths:
+      - Formula/*.rb
+
 jobs:
-  pr-pull:
-    if: contains(github.event.pull_request.labels.*.name, 'pr-pull')
-    runs-on: ubuntu-22.04
+  build-bottles:
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest]
+    runs-on: ${{ matrix.os }}
     steps:
       - name: Set up Homebrew
         uses: Homebrew/actions/setup-homebrew@master
 
+      - run: brew test-bot --only-cleanup-before
```

```suggestion
      - run: brew test-bot --cleanup --only-cleanup-before
```
The help for `--only-cleanup-before` says `Needs --cleanup`...but it is not clear if that option is auto-selected/enabled for you.

---

This line came from `brew tap-new` ([code here](https://github.com/Homebrew/brew/blob/72f7880e2a2d5bd2e1c53b2a425e40b92c50c95c/Library/Homebrew/dev-cmd/tap-new.rb#L96)) and I don't understand what the purpose is...

It definitely seems weird to use `--only-cleanup-before` but never `--cleanup` and never `--only-cleanup-after`... But since I don't understand the code, I don't really want to change it. If anything, I'd just remove it... It seems to work fine without this step.

---

In any case, I'd like to reserve those changes for a future PR if necessary.